### ISSUE TITLE
Update PATH

### ISSUE THREAD

**Issue by Bobbbay** [semissioncontrol/semcOS#14]
The `PATH` by default is bare, it should be more extensive and reach to places like `/usr/local/bin`.

**Comment by Bobbbay** [comment]
Oh, and I almost forgot - this is in `/etc/profile`.

**Comment by Bobbbay** [comment]
[`/etc/profile`](https://github.com/semissioncontrol/semcOS/blob/0.0.3a/slash/x86_64/etc/profile) at `0.0.3a`:

```
export PATH=/bin:/usr/bin

if [ `id -u` -eq 0 ] ; then
        PATH=/bin:/sbin:/usr/bin:/usr/sbin
        unset HISTFILE
fi


# Set up some enironment variables
export USER=`id -un`
export LOGNAME=$USER
export HOSTNAME=`/bin/hostname`
export HISTSIZE=1000
export HISTFILESIZE=1000
export PAGER='/bin/more '
export EDITOR='/bin/vi'
```

**Comment by Bobbbay** [comment]
Fixed at 50e1471. Will be out by `0.0.4a`.
